bwapi/authenticate.py

```diff
@@ -1,12 +1,11 @@
-#!/usr/bin/env python
```

Comment:
Why remove the shbang?

Reply:
When it is install by pip or setuptools, a new shim is created with a hardcoded path to the interpreter and put in the PATH to call the command line function.  Mine is located `/Users/csullivan/.virtualenvs/api_sdk-NSJ8CibS/bin/bwapi-authenticate` and the contents is 
```python
#!/Users/csullivan/.virtualenvs/api_sdk-NSJ8CibS/bin/python3.7
# EASY-INSTALL-ENTRY-SCRIPT: 'bwapi','console_scripts','bwapi-authenticate'
__requires__ = 'bwapi'
import re
import sys
from pkg_resources import load_entry_point

if __name__ == '__main__':
    sys.argv[0] = re.sub(r'(-script\.pyw?|\.exe)?$', '', sys.argv[0])
    sys.exit(
        load_entry_point('bwapi', 'console_scripts', 'bwapi-authenticate')()
    )
```
If I had installed this in my system level python, the command line script is still accessible in my path even in a virtualenvironment.   I think this also has benefits for Windows distributions, though I am not certain.   Other benefits are outlined [here](https://python-packaging.readthedocs.io/en/latest/command-line-scripts.html#the-console-scripts-entry-point)

Reply:
That's neato. Sorry I missed you had made that change, and also never come across it before. Thanks for explaining. :-)